Sources/StreamChatTestTools/CustomUIViewSubclasses.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CustomUIViewSubclasses.swift
+//  Sample
+//
+//  Created by Dominik Bucher on 05.03.2021.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+public enum TestCustomUIKitSubclasses {
+
+    public class TestCustomLabel: UILabel {}
+    public class TestCustomLabel1: UILabel {}
+    public class TestCustomLabel2: UILabel {}
+    public class TestCustomLabel3: UILabel {}
+
+    public class TestCustomView: UIView {}
+    public class TestCustomView1: UIView {}
+    public class TestCustomView2: UIView {}
+    public class TestCustomView3: UIView {}
+    public class TestCustomView4: UIView {}
+    public class TestCustomView5: UIView {}
+
+    public class TestCustomStackView: UIStackView {}
+    public class TestCustomStackView1: UIStackView {}
+    public class TestCustomStackView2: UIStackView {}
+
+    public class TestCustomButton: UIButton {}
+    public class TestCustomButton1: UIButton {}
+    public class TestCustomButton2: UIButton {}
+
+}
```

Comment:
Isn't this extremely annoying to type use? 😅 This should be just a lightweight helper :) It can live inside `StreamChatUITests` and it can literally be just:

```
  class TestLabel: UILabel {}
  class TestLabel1: UILabel {}
  ...
  
  class TestButton: UIButton {}
  class TestButton1: UIButton {}  
  ...
```

so then this
```
customConfig.channelList.itemSubviews.timestampLabel = TestCustomUIKitSubclasses.TestCustomLabel2.self
```
becomes
```
customConfig.channelList.itemSubviews.timestampLabel = TestLabel2.self
```

Reply:
Overengineering for test rules :D I wanted to make it clear that there are test subclasses... But this is actually more simple, so thanks :D